src/playground/service.clj

```diff
@@ -68,8 +69,8 @@
   (session-backend
    {:authfn (fn [request]
               (let [{:keys [username password]} request]
-                (when (= (session.login/password-by-username username) password)
-                  {:username username :password password})))}))
+                (when (= (session.login/password-by-username request username) (hashers/derive password))
```

Comment:
I updated the `session-backend` but I am not sure how this can be used in the login endpoint, other than that it makes no difference to the `authenticated?` as that predicate only checks for a populated `:identity` in the session map.
I don't quite understand how the backend works (looking at the source code)

Reply:
>  I am not sure how this can be used in the login endpoint

I wasn't expecting this, nor I think it's possible